packages/main/test/pages/Toast.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta charset="utf-8">
+	<title>ui5-toast</title>
+
+	<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+	<script src="../../resources/bundle.esm.js" type="module"></script>
+	<script nomodule src="../../resources/bundle.es5.js"></script>
+	<script>delete Document.prototype.adoptedStyleSheets;</script>
+
+	<style>
+		ui5-button, h2 {
+			margin: 1rem;
+		}
+	</style>
+</head>
+
+<body>
+
+	<h2>Toast ( ui5-toast )</h2>
```

Comment:
This is not a big deal, but if you use the <ui5-title> instead of h2, when we switch the test page in High Contrast Black (and the background turns into black -  with the latest master you can observe that), the title would be visible.

Reply:
I meant ui5-title instead of h2